worker.rb

```diff
@@ -30,6 +30,11 @@
   weekly_cleanup_message
 end
 
+#snack czar assumes power at 9am on Monday
+scheduler.cron "0 9 * * 1 #{ENV['TZ']}" do
+  weekly_snack_message
+end
```

Comment:
This isn't accurate to how Snack Czar actually works. The order is typically started well before Monday, but the same Czar who starts the order is also responsible for finishing it (usually at noon on Monday) and receiving it (usually later in the afternoon on Monday, but sometimes on Tuesday if no delivery slots are available). It wouldn't make sense to pick a new Czar on Monday morning, smack in the middle of that process.

Wednesday morning seems like the safest bet to make sure all the lingering duties of the previous Czar are done.